### ISSUE TITLE
Problem: how to add metadata/submission documentation to bags is unclear

### ISSUE THREAD

**Issue by sallain** [archivematica/Issues#209]
**Version of the documentation**
All

**Page (and section, if applicable) where the issue occurs**
https://www.archivematica.org/en/docs/archivematica-1.7/user-manual/transfer/bags/#bags

**Description of the issue**
It's possible to add a metadata directory to a bag in order to carry out various [import metadata](https://www.archivematica.org/en/docs/archivematica-1.7/user-manual/transfer/import-metadata/#import-metadata) workflows. However, the process for doing this is quite particular, and undocumented.

**Suggested fix**
Add the following workflow to the appropriate sections:

1. Make a bag out of the digital objects that you'd like to preserve (I used Bagger). At this point, the bag SHOULD NOT contain the objects you'd like to include in the metadata directory (i.e. DROID outputs, emails, etc.)
2. Save the bag to your desktop.
3. In the top level of the bag, add a metadata directory. Place any metadata into this directory. Archivematica will ignore this directory when verifying the bag.

**Comment by jfcarrano** [comment]
I had figured out the procedure you outline in the ticket and had been using it for a bit but a little while ago I talked to some other Archivematica users who are putting everything, including a metadata/submissionDocumentation folder within the data folder when bagging and that seems to work as well. Access copies from digitization can also be placed in their own folder within the data folder and still be recognized. They made the case that having checksums from the start for all that information wasn't the worst idea. Just FYI for the documentation.

**Comment by sallain** [comment]
Hi @jfcarrano - it's good to know that access copies work! I hadn't gotten around to testing that.

Regarding the metadata/submissionDocumentation folder - you're absolutely right. When I did the testing that resulted in this issue, I was trying to solve a bunch of different bag-related issues, so once I hit on the aforementioned solution as working I didn't look into it any further. In particular, I was looking at using a metadata.csv - and the catch is that you need to amend the filename paths in the CSV to predict where the digital objects will end up once you've created the bag. If I have a transfer that looks like this:

```
CSVmetadata/
├── Landing zone.jpg
├── MARBLES.TGA
└── metadata
    ├── metadata.csv
    └── submissionDocumentation
        └── [datavibe-l]_FW__job_vacancy.rtf
```

My metadata.csv usually looks like this:

```
+---------------------------------------------------------------+
| filename                  | dc.title        | dc.date         |
| objects/Landing zone.jpg  | Landing zone    | July 27, 2009   |
| objects/MARBLES.TGA       | Pretty marbles  | 1988            |
+---------------------------------------------------------------+
```

However, if I know I'm going to be bagging the material, I need to change the filename path to `data/Landing zone.jpg` (or whatever the eventual path may be - there could be an intermediary folder, depending on how you bag the materials). This also applies to using a rights.csv.

Thanks for illuminating another one of the many, many, many ways we can create transfers for Archivematica, Joe!

**Comment by sallain** [comment]
Resolved via artefactual/archivematica-docs#238.
